src/key.rs

```diff
@@ -15,3 +37,111 @@ impl AsMut<[u8]> for Key {
         &mut self.0
     }
 }
+
+impl Key {
+    pub(crate) fn generate_private_key() -> Key {
+        let mut rng = OsRng {};
+        let mut private_key = Key(vec![0_u8; 128]);
+        rng.fill(private_key.as_mut());
+
+        private_key
+    }
+
+    // TODO zeroize
```

Comment:
Is this still a thing?

Reply:
Yes, a bunch of operations which should be zeroized, but I think the entire thing should go away, right now it is copy paste from secret-service-rs

Reply:
But you are already deriving Zerioize here, but in general we would need for things like Secret/(a potential generic Password wrapper)

Reply:
Yes and no. Note that we are turning our safe stuff into biguint and I dont think they take these subtleties into account.